useful workflows/target_workflow_2/target_workflow_2-test.wdl

```diff
@@ -0,0 +1,413 @@
+version 1.0
+
+task align_reads_mock {
+  meta {
+    description: "A mock task"
+  }
+
+  input {
+    File     input_file_1
+    File     input_file_2
+
+    File?    input_file_optional
+
+    String   input_string_default_1 = "this is a default string"
+    String?  input_string_optional
+    Boolean? input_bool_default = false
+
+    Int?     input_int_optional
+    String   input_string_default_2 = "my-docker-image.123"
+
+    String   input_string_default_3 = "this is a second default string"
+  }
+
+  Float variable_float = 0.67
+  Int variable_int = 5
+
+  command {
+    echo "align_reads_mock"
+  }
+
+  output {
+    File   output_file_1   = input_file_1
+    File   output_file_2   = input_file_1
+    File   output_file_3   = input_file_1
+    File   output_file_4   = input_file_1
+    File   output_file_5   = input_file_1
+    File   output_file_6   = input_file_1
+    File   output_file_7   = input_file_1
+    Int    output_int_1    = variable_int
+    Int    output_int_2    = variable_int
+    Int    output_int_3    = variable_int
+    Float  output_float_1  = variable_float
+    Float  output_float_2  = variable_float
+    Int    output_int_4    = variable_int
+    Int    output_int_5    = variable_int
+    String output_string_1 = input_string_default_1
+    String output_string_2 = input_string_default_2
+  }
+}
+
+task ivar_trim_mock {
+  meta {
+    description: "A mock task"
+  }
+
+  input {
+    File   input_file
+    File?  input_file_optional
+    Int?   input_int_optional_1
+    Int?   input_int_optional_2
+    Int?   input_int_default = 1
+    Int?   input_int_optional_3
+    
+    Int?   input_int_optional_4
+    String input_string_default = "ivar trim mock string"
+  }
+
+  Float variable_float = 0.82
+
+  command {
+    echo "ivar_trim_mock"
+  }
+
+  output {
+    File   output_file   = input_file
```

Comment:
It might be best to use files produced IN the task rather than mirroring inputs. 

1. Tests that Cromwell is localizing and delocalizing properly
2. Cromwell is a little strange about mirrored input files. I have a feeling it treats them differently than generated files (eg it just reuses the old path that came in as an input rather than generating a new path). Not sure I remember the exact problem but I do have a vague sense of "here be dragons"